lib/extensions.js

```diff
@@ -61,6 +61,9 @@ module.exports = function (realmConstructor) {
   realmConstructor._UUID = realmConstructor.BSON.UUID;
   const { DefaultNetworkTransport } = require("@realm/network-transport");
   realmConstructor._networkTransport = new DefaultNetworkTransport();
+  realmConstructor._defaultOnDiscardCallback = (_, path) => console.log(`ClientReset onDiscard: path = ${path}`);
+  realmConstructor._defaultOnRecoveryCallback = (beforeRealm, afterRealm) =>
+    console.log(`ClientReset onRecovery: before path = ${beforeRealm.path}, after path = ${afterRealm.path}`);
```

Comment:
Or perhaps, just don't do anything at all?

Reply:
@takameyer You are thinking the same, right?

Reply:
Yeah, I was also thinking we could just do nothing there.  However, if we defaulted a `console.error` for sync errors, then it would also be visible that a client reset occurred.

Reply:
It looks like @takameyer is in favor of the same.